UE4SS/include/GUI/GUI.hpp

```diff
@@ -213,6 +240,16 @@ namespace RC::GUI
         {
             return m_console;
         };
+
+        auto get_gfx_backend() -> std::unique_ptr<GfxBackendBase>&
+        {
+            return m_gfx_backend;
+        };
+
+        auto get_os_backend() -> std::unique_ptr<OSBackendBase>&
+        {
+            return m_os_backend;
+        };
```

Comment:
```suggestion
        auto get_gfx_backend() -> GfxBackendBase&
        {
            return *m_gfx_backend;
        };

        auto get_os_backend() -> OSBackendBase&
        {
            return *m_os_backend;
        };
```
You shouldn't return `std::unique_ptr<T>` unless you intend to move ownership of the pointer which clearly isn't the goal of these two functions.
You should instead return a reference to the underlying object.

I also don't see any usages for either of these two functions which means they should be removed unless you plan on using them in this PR later on.

Reply:
Earlier experiment I forgot to remove, removed now, thank you